hbase-server/src/test/java/org/apache/hadoop/hbase/master/procedure/TestSCPBase.java

```diff
@@ -141,36 +135,6 @@ protected long getSCPProcId(ProcedureExecutor<?> procExec) {
     return procExec.getActiveProcIds().stream().mapToLong(Long::longValue).min().getAsLong();
   }
 
-  private void assertReplicaDistributed(Table t) throws IOException {
```

Comment:
This you will come back later after the other new JIRA for load balancer is fixed?

Reply:
Yes, add it back in HBASE-23078.